projects/user_publication/publications.py

```diff
@@ -28,6 +33,11 @@ def import_pubs(dry_run=True, file_name="", source="all"):
 
     # Process each publication
     for source, pub in pubs:
+        same_pubs = utils.get_publications_with_same_attributes(pub, Publication)
+        if same_pubs.exists():

```

Comment:
This check is unnecessary. A loop through an empty `QuerySet` is a no-op.

Reply:
I would like to skip the publication if there is at least a single publication with the same attributes after updating all the same publications so need that check.